.github/workflows/docs.yml

```
@@ -0,0 +1,31 @@
+  workflow_dispatch: {}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pages: write
+      id-token: write
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.10)

[error] property "deployment" is not defined in object type {}

(expression)